tests/testthat/test-say.R

```diff
@@ -9,6 +9,21 @@ test_that("say types works as expected", {
   
   # expect string on type=string
   expect_is(say(type = "string"), "character")
+  
+  expect_equal(
```

Comment:
Perhaps this instead:

```r
expect_equal(
  length(suppressMessages(say("foo", by_color = "cyan"))) + 1,
  length(suppressWarnings(say("foo", type = "warning")))
)
```

so that we don't get the stuff printed out in the test run

Reply:
Oh for sure. This test isn't even totally necessary so I'd be fine with scrapping it if you'd prefer.